sdk/core/azure-core-amqp/samples/internal/eventhub_get_properties_sample/eventhub_get_eventhub_properties_sample.cpp

```diff
@@ -1,10 +1,10 @@
 // Copyright (c) Microsoft Corporation.
```

Comment:
BTW, I think `internal/samples` directory would be better than `samples/internal`. I am not suggesting to necessarily fix in this PR.

Reply:
samples/internal was chosen because the hierarchy of folders in the repo is:

- sdk/<service>/<package>
  - inc
  - samples
  - src
  - tests
  - vcpkg
 
And who knows - there might be samples for AMQP message or AMQP value created.

Reply:
#5091